packages/algolia/test/client.test.ts

```diff
@@ -138,20 +138,38 @@ describe('Algolia Search Client', () => {
     expect(algoliaSearchIndex.search).toBeCalledWith('query', {
       hitsPerPage: 10,
       page: 0,
-      filters: 'some-filters AND __meta.type:"research-output"',
+      filters: 'some-filters AND (__meta.type:"research-output")',
```

Comment:
its worth adding a test where `some-filters` is a list with an `OR` - that will show that we need brackets for that part too